tests/conftest.py

```diff
@@ -71,7 +71,7 @@ def grant_test_db(writer_uri, uri_db_reader):
     db_reader = urlparse(uri_db_reader).username
     db_reader_password = urlparse(uri_db_reader).password
 
-    postgres_db = sqlalchemy.create_engine(writer_uri, echo=False, isolation_level="AUTOCOMMIT", client_encoding="utf8")
+    postgres_db = sqlalchemy.create_engine(writer_uri, echo=False, isolation_level="SERIALIZABLE", client_encoding="utf8")
```

Comment:
I am not sure about this. I might revert this, especially if devs complain in this PR.

Reply:
Did that help with running the tests?

Reply:
I reverted back in the end.